src/TrigScint/Event/TrigScintQIEDigis.cxx

```diff
@@ -0,0 +1,48 @@
+/**
+ * @file TrigScintQIEDigis.cxx
+ * @brief class for storing QIE output
+ * @author Niramay Gogate, Texas Tech University
+ */
+
+#include "TrigScint/Event/TrigScintQIEDigis.h"
+#include <iostream>
+#include <exception>
+ClassImp(ldmx::TrigScintQIEDigis);
+
+namespace ldmx {
```

Comment:
Are we going to get in trouble here, or are we moving to the new namespace logic (`trigscint` rather than `ldmx`) later on?

Reply:
yes, this has been fixed in the latest commit